contrib/babelfishpg_tsql/src/multidb.c

```diff
@@ -1414,6 +1428,21 @@ get_guest_role_name(const char *dbname)
 	}
 }
 
+char *
+get_db_accessadmin_role_name(const char *dbname)
+{
+	char	   *name = palloc0(MAX_BBF_NAMEDATALEND);
+
+	if (get_migration_mode() == SINGLE_DB && strcmp(dbname, "master") != 0
+	    && strcmp(dbname, "tempdb") != 0 && strcmp(dbname, "msdb") != 0)
```

Comment:
Looks like we need to a utility function called `is_builtin_db(const char *dbname)` to check if it is one of master/tempdb/msdb.

Reply:
We change this to a macro

Reply:
Done